content/en/docs/installing-on-linux/public-cloud/install-ks-on-azure-vms.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Deploy KubeSphere on Azure VM Instance"
+keywords: "Kubesphere, Installation, HA, high availability, load balancer, Azure"
+description: "The tutorial is for installing a high-availability cluster on Azure."
+---
+
+## Before you begin
+
+Technically, you can either install, administer, and manage Kubernetes yourself or go for a managed Kubernetes solution. If you are looking for a way to take advantage of Kubernetes with a hands-off approach, a fully managed platform solution is what you’re looking for, please see [Deploy KubeSphere on AKS](../../../installing-on-kubernetes/hosted-kubernetes/install-ks-on-aks) for more details. But if you want a bit more control over your configuration and setup a Highly Available clusters on Azure, this instruction will help you to setup a production-ready Kubernetes and KubeSphere.
+
+## Introduction
+
+In this tutorial, we will use two key features of Azure virtual machines (VMs): 
+
+- Virtual Machine Scale Sets: Azure VMSS let you create and manage a group of load balanced VMs. The number of VM instances can automatically increase or decrease in response to demand or a defined schedule(Kubernates Autoscaler is available, but not covered in this tutorial, see [autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/azure) for more details), which perfectly fits the Worker Nodes.
+- Availability sets: An availability set is a logical grouping of VMs within a datacenter that automatically distributed across fault domains. This approach limits the impact of potential physical hardware failures, network outages, or power interruptions. All the Master and ETCD VMs will be placed in an Availability sets to meet our High Availability goals.
+
+Besides those VMs, other resources like Load Balancer, Virtual Network and Network Security Group will be involved.
+
+## Prerequisites
+
+- You need an [Azure](https://portal.azure.com) account to create all the resources.
+- Basic knowledge of [Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/)(ARM) templates, which are files that define the Azure infrastructure and configuration.
+- Considering data persistence, for a production environment, we recommend you to prepare persistent storage and create a StorageClass in advance. For development and testing, you can use the integrated OpenEBS to provision LocalPV as the storage service directly.
+
+## Architecture
+
+Six machines of "Ubuntu 18.04" will be deployed in Azure Resources Group. Tree of them are grouped into an Availability sets, which will be used by Master and ETCD. Another tree VMs will be defined as a VMSS, Worker nodes will be run on it.
```

Comment:
Tree? I guess it is `three` here   :)

Reply:
which will be used by Master and ETCD. => playing the role of both Master and ETCD of the Kubernetes control plane.

Reply:
which will be used by Master and ETCD. => playing the role of both Master and ETCD of the Kubernetes control plane.